apps/expo/package.json

```diff
@@ -12,6 +12,7 @@
     "type-check": "tsc --noEmit"
   },
   "dependencies": {
+    "@acme/tailwind-config": "^0.1.0",
```

Comment:
Required?

Reply:
Good point, :) 
It shouldn't be since it's only for the build, but I got typescript errors earlier and I saw that it was a `dependency` in `@acme/nextjs` as well so I thought there was something I don't fully understand yet.

However when I ran `pnpm run clean`, installed everything and reset TS server I realize it was a false TS error and everything runs fine with `@acme/tailwind-config` as a `devDependency`.

PR updated